README.md

```diff
@@ -97,6 +98,10 @@ deface file.
 ```
 
 When the Square hosted checkout finish, Square will redirect you automatically to the redirect URL given in the preferences of the `Square` payment method.
+
+Uncomment the `config.square_payment_method = Spree::PaymentMethod.find_by(ENV['SQUARE_PAYMENT_METHOD'])` line in `config/initializers/solidus_square.rb` file,
```

Comment:
I see it yesterday, we have to change:

`Spree::PaymentMethod.find_by` to `Spree::PaymentMethod.find`
and
`SQUARE_PAYMENT_METHOD` to `SQUARE_PAYMENT_METHOD_ID`
It requires the same changes on the [template](https://github.com/nebulab/solidus_square/blob/main/lib/generators/solidus_square/install/templates/initializer.rb#L7).